trillian/ctfe/configpb/config.proto

```diff
@@ -154,3 +182,26 @@ message SignedTreeHead {
   bytes sha256_root_hash = 3;
   bytes tree_head_signature = 4;
 }
+
+// Cache describes the cache type and configurations.
+//
+// NEXT_ID: 5
+message Cache {
+  // CacheType represents the underlying cache type implementation.
+  // If unspecified, no cache is used.
+  enum CacheType {
+    CACHE_TYPE_NONE = 0;
+    CACHE_TYPE_LRU = 1;
+  }
+  CacheType type = 1;
+
+  // Cache size 0 means unlimited size. Default is 0.
```

Comment:
I think this is fine because you computed that the total size of all the chains is O(MB), right?  In practice, putting a reasonably high value is a good safety mechanism I'd say.

Reply:
Correct. The estimated size of certificate chains in each log is around 120 MB.

This `Cache` message is designed not only for non-leaf certificate chain cache, so I've added `// Setting the cache size (if available in the cache type) avoids the unexpected certificate chain growth rate.` in the comment of `Cache extra_data_non_leaf_certificate_chain_cache`.